extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -314,7 +314,7 @@ export class AmpStoryPage extends AMP.BaseElement {
 
   /** @private */
   markPageAsLoaded_() {
-    dispatch(this.element, EventType.PAGE_LOADED, true);
+    dispatch(this.win, this.element, EventType.PAGE_LOADED, true);
```

Comment:
The fourth parameter `true` used to be `opt_bubbles` but is now `opt_payload`.

Also, `opt_bubbles` no longer exist, you now need to send `opt_eventInit`, which would give something like `{bubbles: true}` instead of just `true`.

Should be:
````
  dispatch(this.win, this.element, EventType.PAGE_LOADED, undefined, {bubbles: true});
````

Reply:
Ahh I see! Sorry for the dumb mistake haha